Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Build cadence binaries
+FROM golang:1.12.7-alpine AS builder
+
+RUN apk add --update --no-cache ca-certificates make git curl mercurial bzr
+
+# Install dep
+ENV DEP_VERSION=0.5.4
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | INSTALL_DIRECTORY=/usr/local/bin DEP_RELEASE_TAG=v${DEP_VERSION} sh
```

Comment:
we always use the latest dep: https://github.com/uber/cadence/blob/master/install-dep.sh

Reply:
That's generally not a good practice if you want to ensure repeatable builds.

Reply:
That's fine. I think what we decided is always using latest one, if it doesn't build, we fix it right away.